plugins/modules/site_workflow_manager.py

```diff
@@ -610,8 +630,29 @@ def site_exists(self):
         site_exists = False
         current_site = {}
 
-        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.5.3") <= 0:
+        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") >= 0:
+            sites = None
+            response = self.get_site(site_name_hierarchy)
+            self.log("Raw response from get_site: {}".format(response), "DEBUG")
+            if isinstance(response, dict):
+                sites = response.get("response", [])
```

Comment:
you can do
all the if sites check and for loop here...

Reply:
Modified